niri-config/src/lib.rs

```diff
@@ -2983,14 +2997,16 @@ mod tests {
             }
 
             binds {
+                Mod+Escape { toggle-keyboard-shortcuts-inhibit; }
+                Mod+Shift+Escape allow-inhibiting=true { toggle-keyboard-shortcuts-inhibit; }
                 Mod+T allow-when-locked=true { spawn "alacritty"; }
                 Mod+Q { close-window; }
                 Mod+Shift+H { focus-monitor-left; }
                 Mod+Ctrl+Shift+L { move-window-to-monitor-right; }
                 Mod+Comma { consume-window-into-column; }
                 Mod+1 { focus-workspace 1; }
                 Mod+Shift+1 { focus-workspace "workspace-1"; }
-                Mod+Shift+E { quit skip-confirmation=true; }
+                Mod+Shift+E allow-inhibiting=false { quit skip-confirmation=true; }
```

Comment:
I wonder if `allow-inhibiting=false` is a bit hard to wrap one's head around? Maybe something along `prevent-inhibiting=true` would be better, human config file reading and writing wise?

Reply:
i don't feel like that change is an improvement, because `prevent-inhibiting=true` is a double negative. but also, i agree with your concern, and i think it's slightly confusing because `allow-inhibiting=false` is also a double negative, because the value it will be set to is false.

is there a way to name it that isn't a double negative? `always-available=true`, but more descriptive? borrow an idiom from CSS and call it `important=true`? maybe with the exclamation point to drive home the correlation? `!important=true`. though it reads as "not".

Reply:
Hmmm, difficult question indeed. I like the word "inhibit" being there because it's the technical term for this (unless it isn't always? do some apps re-phrase it?).

If we drop inhibit, maybe something like "always-accessible" is better than "always-available"? That's kind of still not very descriptive, while it does control a very specific thing, so it would be good if it were descriptive...